avocado_vt/plugins/vt_cluster.py

```
@@ -0,0 +1,246 @@
+
+        This method ensures that both the cluster manager and the individual
+        nodes are cleaned up. It is designed to be robust, attempting all
+        cleanup steps even if some fail.
+
+        :param job: The Avocado job object.
+        :type job: avocado.core.job.Job
+        """
+        if cluster.get_all_nodes():
+            try:
+                self._cleanup_mgr()
+            except Exception as err:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)